configurations/tablets_multidc_multirack.yaml

```diff
@@ -1,2 +1,3 @@
 n_db_nodes: '4 4'
 simulated_racks: 4
+rack_aware_loader: false
```

Comment:
in this case it should be True to keep the old behaviour

Reply:
fixed